src/Microsoft.AspNet.Mvc.Core/ModelBinding/Validation/DefaultComplexObjectValidationStrategy.cs

```diff
@@ -12,6 +13,8 @@ namespace Microsoft.AspNet.Mvc.ModelBinding.Validation
     /// </summary>
     public class DefaultComplexObjectValidationStrategy : IValidationStrategy
     {
+        private static readonly bool IsMono = Type.GetType("Mono.Runtime") != null;
```

Comment:
The `IsMono` field in `SymbolsUtility` is `Lazy<bool>` but it's unlikely someone would use MVC without validation. Not sure whether that optimization is useful in `SymbolsUtility` or necessary here. Thoughts?